tests/testapps/CTCatalog/CTCatalog/CTCBaseViewController.h

```diff
@@ -17,6 +17,19 @@
 #import <UIKit/UIKit.h>
 #import <CoreText/CoreText.h>
 
+// Convenience macro to create and add rows for unimplemented methods to the table
+#define ADD_UNIMPLEMENTED(ARRAY, METHOD_NAME) [ARRAY addObject:createTextCell(METHOD_NAME, @"UNIMPLEMENTED")]
+
+// Creates a table cell with title and text value
+static UITableViewCell* createTextCell(NSString* title, NSString* value) {
```

Comment:
>static [](start = 0, length = 6)

static methods in headers don't make much sense. you could make it inline, or just have the implementation in the mm file and declaration here.